### ISSUE TITLE
Create as full method signature as possible with parameters

### ISSUE THREAD

**Issue by LenarBad** [LenarBad/EasyLog#1]
https://github.com/LenarBad/EasyLog/blob/4824c3ec56d84b889cc0bf3d018ba14d81a021dd/src/main/java/io/lenar/easy/log/EasyLogger.java#L50

**Comment by LenarBad** [comment]
We need to log the full method signature like:
```java
com.my.package.MyClass.myMethod(SomeType parameter1, AnotherType parameter)
```
instead of just <code>myMethod</code>

**Comment by LenarBad** [comment]
Implemented as part of #14
